src/apps/explorer/pages/Home/index.tsx

```diff
@@ -18,14 +18,13 @@ const Wrapper = styled.div`
     font-weight: ${({ theme }): string => theme.fontBold};
     width: 100%;
     margin: 0 0 2.4rem;
-    font-size: 4.2rem;
+    font-size: 2rem;
     line-height: 1;
   }
 
   ${media.mobile} {
     > h1 {
       line-height: 1.2;
-      font-size: 3.2rem;
```

Comment:
font-size: 1.70rem;
(decrease the font-size on mobile)

Reply:
yes, done, thanks!